xrdp/xrdp_mm.c

```diff
@@ -1042,27 +1042,110 @@ dynamic_monitor_data_first(intptr_t id, int chan_id, char *data, int bytes,
     return 0;
 }
 
+/******************************************************************************/
+static int
+process_dynamic_monitor_description(struct xrdp_wm *wm, struct display_size_description *description) {
```

Comment:
Coding standard again, also a few lines below.

Reply:
Fixed with merge to astyle.